integrationtests/singlevm_key_ed25519/main.tf

```diff
@@ -11,20 +11,20 @@ terraform {
   }
 }
 
-provider "grid" {
-  key_type = "ed25519"
-}
+# provider "grid" {
```

Comment:
let's not modify any of the main.tf files in this pr. this may be needed in the future